api/TaskTracker/TaskTracker.Data/Entities/TaskEntity.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaskTracker.Data.Entities.Enums;
+using TaskStatus = TaskTracker.Data.Entities.Enums.TaskStatus;
+
+namespace TaskTracker.Data.Entities;
+public class TaskEntity
+{
+    public int Id { get; set; }
+    [Required]
+    public string Title { get; set; } = string.Empty;
```

Comment:
do not initialize with string.Empty. I guess, the Title is required, so you can write it just like a: 
public required string Title { get; set; }

If we are talking about Description fields, it may be optional, so change it to nullable string: 
public string? Description { get; set; }